tox.ini

```diff
@@ -11,24 +20,34 @@ wheel_build_env = .pkg
 
 [testenv:coverage]
 deps = coverage
-basepython = python3.12
+base_python = python3.12
 commands =
-    coverage run --source={envsitepackagesdir}/pytest_check,tests -m pytest 
+    coverage run --source={envsitepackagesdir}/pytest_check,tests -m pytest
     coverage report --fail-under=100 --show-missing
 description = Run pytest, with coverage
 
 [testenv:pytest_earliest]
 deps = pytest==7.0.0
-basepython = python3.11
+base_python = python3.11
 commands = pytest {posargs}
 description = Run earliest supported pytest
 
 [testenv:lint]
-skip_install = true
-deps = ruff
-basepython = python3.12
-commands = ruff check src tests examples
-description = Run ruff over src, test, exampless
+deps =
+    mypy
+    ruff
+base_python = python3.12
+commands =
+    ruff check src tests examples
+    mypy --strict --pretty src
+    mypy --pretty tests
```

Comment:
I decided to add mypy to the linting step rather than creating a separate one. 🤷